OpenScholar/azuredeploy.json

```diff
@@ -0,0 +1,201 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminVMUsername": {
+      "type": "string",
+      "minLength": 3,
+      "metadata": {
+        "description": "User name for the Virtual Machine."
+      }
+    },
+    "adminVMPassword": {
+      "type": "securestring",
+      "minLength": 3,
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "dnsNameForPublicIP": {
+      "type": "string",
+      "minLength": 3,
+      "defaultValue": "[concat('os', uniqueString(resourceGroup().id))]",
+      "metadata": {
+        "description": "Globally unique DNS Name for the Public IP used to access the Virtual Machine."
+      }
+    },
+    "databaseName": {
+      "type": "string",
+      "minLength": 3,
+      "defaultValue": "openscholarDB",
+      "metadata": {
+        "description": "Name for the Database."
+      }
+    },
+    "mysqlPassword": {
+      "type": "securestring",
+      "minLength": 3,
+      "metadata": {
+        "description": "Provide Password for the My SQL."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    }
+  },
+  "variables": {
+    "imagePublisher": "Canonical",
+    "imageOffer": "UbuntuServer",
+    "nicName": "scholarVMNic",
+    "addressPrefix": "10.0.0.0/16",
+    "subnetName": "Subnet",
+    "subnetPrefix": "10.0.0.0/24",
+    "publicIPAddressType": "Dynamic",
+    "vmSize": "Standard_D2_v2",
+    "vmName": "openscholarVM",
+    "virtualNetworkName": "scholarVNET",
+    "subnetRef": "[concat(resourceId('Microsoft.Network/virtualNetworks', variables('virtualNetworkName')), '/subnets/', variables('subnetName'))]",
+    "installScriptName": "openDrupal.sh",
+    "installScriptUri": "[concat('https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/OpenScholar/', variables('installScriptName'))]",
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts